.github/workflows/publish_release.yml

```diff
@@ -40,7 +43,7 @@ jobs:
         uses: docker/login-action@v3
         with:
           registry: ghcr.io
-          username: ${{ github.repository_owner }}
+          username: ${{ github.actor }}
```

Comment:
Does it matter who we login as anymore?

Reply:
Not really since that workflow should be always be run from the cyclus repository.  However, I think logging in as `github.actor` is more transparent as far as what permissions exist (and the example in the [action docs](https://github.com/docker/login-action?tab=readme-ov-file#github-container-registry) uses `github.actor`)